### PR TITLE
1.0.0 changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
Fixed typo, grammatical, and formatting issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 # Overview
 
-This solution pack helps community users to submit their custom solution to FSR team.
+The **Submit Your Content** solution pack empowers community members to seamlessly submit their solutions for review on FortiSOAR’s Content Hub. By launching their FortiSOAR instance and leveraging playbooks, users can streamline the submission process, ensuring efficient handling of their valuable contributions.
```
Comment (cs-suyog-jinturkar) [inline README.md:10]: This section is very important and current details are not sufficient. This is an entry point to start using this solution. Please elaborate more on usecases, benefits in details, adding a widget screenshot, widget link, etc.

File: docs/setup.md [hunk 2]
```diff
@@ -3,14 +3,15 @@
...
+2. From the list of solution pack that appears, search for **Submit Your Content**.    
 3. Click the **Submit Your Content** solution pack card.   
-4. Click the **Install** button on the bottom to begin the installation.
+4. Click the **Install** button on the lower part of the screen to begin the installation.
 
 # Configuration
 
-By default SMTP is selected as a mode to send email. Can be changed through playbook, send email.
-<img src="./res/configuration.png" alt="Access Control" style="border: 1px solid #A9A9A9; border-radius: 4px; padding: 10px; display: block; margin-left: auto; margin-right: auto; width: 80%">
+The SMTP connector is configured to send email, by default. You can change your preference, on the connector to use for sending emails, by editing the playbook **Send Email**.
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md]: Please also mention PB collection and steps to be modified

File: docs/usage.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 | [Home](../README.md) |
 |----------------------|
 
+# Usage
+
+For detailed usage refer to the Submit Content Form widget.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: Widget document link will be useful here

File: README.md (hunk 1 above)
Comment (cs-yash-patodkar) [inline README.md:10]: _By launching their FortiSOAR instance and leveraging playbooks, users can streamline the submission process, ensuring efficient handling of their valuable contributions._

I dont think this is required can be modified to something like users can upload their cusstom solutions like connector widgets and sp's

File: README.md [hunk 4]
```diff
@@ -7,7 +7,13 @@
...
+
+This solution pack simplifies content submission by breaking down the entire process into manageable steps addressed through user-friendly interfaces.
+
+- **Integrated with FortiSOAR**: Content submission is seamlessly integrated with FortiSOAR. You can use the [FortiSOAR RDK](https://docs.fortinet.com/fortisoar/connectors/rapiddevkit) to create solution packs, widgets, or connectors directly from the FortiSOAR interface. Once you've created your content, use the **Submit Your Content** solution pack to submit it to the FortiSOAR Integrations team for review and subsequent upload to the Content Hub.
```
Comment (cs-suyog-jinturkar) [inline README.md]: to create solution packs, widgets, or connectors directly from the FortiSOAR interface ---> we don't create widget and sp using rdk. Also, rdk is not used from the FSR interface.

File: README.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply README.md:10]: Added more details

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**